src/components/auth/Signup.jsx

```diff
@@ -37,17 +99,46 @@ export function Signup() {
 
           <div className="flex-row spc-btwn">
             <label>
-              <input type="checkbox" /> I accept all terms & conditions
+              <input
+                type="checkbox"
+                onChange={(e) => {
+                  signupDispatch({ type: "TAC" });
+                }}
+              />
+              I accept all terms & conditions
             </label>
           </div>
-          <button className="cart-btn full-width">Create New Account</button>
+          <button
+            className="cart-btn full-width"
+            onClick={async () => {
+              console.log(
+                signupInfo.firstName,
+                signupInfo.lastName,
+                signupInfo.email,
+                signupInfo.password,
+                signupInfo.confirmPwd,
+                signupInfo.tAc
```

Comment:
you can use console.table(signupInfo) instead would save you the hustle of debugging.

Reply:
I forgot to remove that after debugging thanks for pointing out.